libvast_test/src/symbols.cpp

```diff
@@ -0,0 +1,21 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#define CAF_TEST_NO_MAIN
+#include "caf/test/unit_test_impl.hpp"
```

Comment:
Why is this file including the impl header?

Reply:
Several fixture implementations call into the functions defined in `unit_test_impl.hpp`, when linking they have to be available.

Reply:
In other words: they are missing in `libcaf_test`?

Reply:
libcaf_test is not really a library right now, meaning there is no target which goes by that name. I have a rewrite of cafs cmake code in progress [1], but I need to figure out a good way to support old cmake versions before I can submit it. Once this is done the `impl` code can move into a regular library.

[[1] https://github.com/tobim/actor-framework/tree/modernize-cmake](https://github.com/tobim/actor-framework/tree/modernize-cmake)

Reply:
Ok. If `libcaf_test` is header-only at the moment than let's go with this approach.